### ISSUE TITLE
Issue: All digits replaced with spelled-out versions

### ISSUE THREAD

**Issue by OverloadUT** [matt-kruse/alexa-app#45]
I appears that the SSML update to this app introduced an issue where alexa-app is transforming all digits (1, 2, 3) in to the spelled-out version (one, two, three).

It also makes it impossible to use the "say as digits" feature in SSML due to the digits never actually ending up in the string.

**Comment by rickwargo** [comment]
Greg, that was done intentionally on a misunderstanding of the Alexa "rules." I am looking for that rule in Alexa docs and cannot find. For a work around, the following lines of code can be removed from to-ssml.js:

```javascript
var Numbered = require('numbered');
```
and

```javascript
            if (str.match(/[0-9]/)) {
                str = str.replace(/[0-9]+(.[0-9])?(?![^<]+>)/g, function (num) {
                    return Numbered.stringify(num).replace(/-/g, ' ');
                });
            }
```

I will submit a PR to resolve. Thanks for pointing that out.

**Comment by dblock** [comment]
Closed via #47.
